src/init/index.ts

```diff
@@ -29,6 +29,7 @@ const featureFns = new Map<string, (setup: any, config: any, options?: any) => P
   ["project", features.project], // always runs, sets up .firebaserc
   ["remoteconfig", features.remoteconfig],
   ["hosting:github", features.hostingGithub],
+  ["frameworkstack", features.frameworkstack],
```

Comment:
This chioce has to be inside an expeirment. We don't want this choice to show up for most users.

Reply:
done